flow/scripts/synth_canonicalize.tcl

```diff
@@ -2,5 +2,11 @@ source $::env(SCRIPTS_DIR)/synth_preamble.tcl
 hierarchy -check -top $::env(DESIGN_NAME)
 # Get rid of unused modules
 opt_clean -purge
+
+if {$::env(SYNTH_STRIP)} {
+  puts "Stripping source code information"
+  attrmap -remove \src
```

Comment:
`setattr -unset src` might be a better command for this since it doesn't log each attribute instance it removes. Also you need `setattr -mod -unset src` to strip the attribute on modules. With `attrmap` second command is needed too